examples/fault-injection/Dockerfile-backend

```diff
@@ -0,0 +1,6 @@
+FROM python:3.7-alpine3.8
```

Comment:
I'm not sure if you really need those Dockerfiles, perhaps you can just use existing `httpbin` docker image for service and `curl` for client?

Reply:
Good point. I changed to use httpbin docker image: https://hub.docker.com/r/kennethreitz/httpbin/